cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -10,16 +11,36 @@
 logger = logging.getLogger(__name__)
 
 
+class RecipeType(Enum):
+    ROLLOUT = "rollout"
+    EXPERIMENT = "experiment"
+    EMPTY = ""
+
+
 class RemoteSettings:
     def __init__(self, sdk: SDK):
-        self.recipes: Dict[str, List[Any]] = {"data": []}
+        self.recipes: dict[str, list[Any]] = {"data": []}
         self.url: str = remote_setting_url
         self.sdk = sdk
 
-    def get_recipes(self) -> Dict[str, List[Any]]:
+    def get_recipes(self) -> dict[str, list[Any]]:
         return self.recipes
 
-    def update_recipes(self, new_recipes: Dict[str, List[Any]]) -> None:
+    def get_recipe_type(self, experiment_slug: str) -> str:
+        recipes_data = self.get_recipes()["data"]
+        if not recipes_data:
+            return RecipeType.EMPTY.value
```

Comment:
Will this ever be true? Is there a change ```recipes_data``` returns ```False``` ?

Reply:
I'm dumb, it's a list

Reply:
You're good I was literally about to pull up a python console to check how it interacts with lists when casting to bool